bedrock/base/templates/product-all-unified-macros.html

```diff
@@ -0,0 +1,98 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% macro select_product_list(products, disabled=False) %}
+ <select id="select-product" class="c-selection-input" {% if disabled %}disabled{% endif %}>
```

Comment:
NIT: Include the space in the `if` block to avoid superfluous space at the end of the `select` tag.